lib/verbalize/build.rb

```diff
@@ -0,0 +1,60 @@
+module Verbalize
+  class Build
+    class << self
+      def call(required_keywords = [], optional_keywords = [])
```

Comment:
since there is only a single class method here, IMO the general preference would be to use `self.call` syntax instead of a `class << self` block.